App.test.js

```diff
@@ -0,0 +1,58 @@
+import { render, screen, fireEvent } from '@testing-library/vue'
```

Comment:
this file is wrong. it should be in component folder and it should have name button.test.js

Reply:
done and moved to component folder